extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -4,7 +4,7 @@ define('ooyala-player', function () {
 	// TODO ooyala only supports font icons so we probably need to extract our DS icons to font
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)